src/browser/api/browser_view.ts

```diff
@@ -35,31 +38,44 @@ export async function create(options: BrowserViewOpts) {
         setBounds(ofView, options.bounds);
     }
 }
-
+export function hide(ofView: OfView) {
```

Comment:
Not sure I am a fan of this. why not simply remove the browserview from the render process directly.